friends/src/Components/Home/Home.css

```diff
@@ -0,0 +1,39 @@
+body {
+  /* margin: 0 auto;
+  height: 100%;
+  width: 100%; */
+  background-image: url("./img/background.jpg");
```

Comment:
Comment this background image here, otherwise it will override the whole page's background.